sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLChangeLogDAO.java

```diff
@@ -9,11 +9,13 @@
 
 import uk.ac.bbsrc.tgac.miso.core.data.ChangeLog;
 import uk.ac.bbsrc.tgac.miso.core.store.ChangeLogStore;
+import uk.ac.bbsrc.tgac.miso.core.util.CoverageIgnore;
 
 public class SQLChangeLogDAO implements ChangeLogStore {
   private static class ChangeLogMapper implements RowMapper<ChangeLog> {
 
     @Override
+    @CoverageIgnore
```

Comment:
cool, I've been ignoring the mapping functions too as hibernate will take care of that. :white_check_mark: